app/next-client-app/app/globals.css

```diff
@@ -1,3 +1,10 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+@layer base {
+    input[type="number"]::-webkit-inner-spin-button,
```

Comment:
What does this do?

Reply:
It gets rid of this 
<img width="367" alt="Screenshot 2024-05-17 at 10 35 00" src="https://github.com/Health-Informatics-UoN/CaRROT-Mapper/assets/94780632/5cea994e-ead3-4634-8584-4017039620f0">